tfjs-backend-webgpu/src/backend_webgpu_test.ts

```diff
@@ -214,6 +214,80 @@ describeWebGPU('backend webgpu', () => {
     // Now that data has been downloaded to the CPU, dataSync should work.
     expect(() => c.dataSync()).not.toThrow();
   });
+
+  it('should reuse textures when fromPixels have same input size', async () => {
```

Comment:
You need to make sure that `WEBGPU_USE_IMPORT` is false. Otherwise, for the first video case, `expect(freeTexturesAfterFromPixel).toEqual(1);` is 0 not 1.

Reply:
done